### DISCUSSION TITLE
Request : add environment variable for CLIENT_VERSION

### DISCUSSION THREAD

**Discussion by guedz** [drakkan/sftpgo#1107] [Ideas]
Hello,

Is it possible to add a "client version" to a hook environment variable to identify if it is (for example) a manual connection (a Filezilla client) or via an automated script (example ncftp)?

Example of data on general log file of sftpgo : client version \"SSH-2.0-FileZilla_3.60.2\"

Best regards and thanks again for your work/assistance.

**Comment by drakkan** [comment]
Hello,

the [support policy](https://github.com/drakkan/sftpgo#support-policy) changed. 
I no longer implement anything that makes sense for free, sorry.
If people just ask and never give back, the project will die.
You need at least a basic sponsorship level to get support, thanks for understanding

**Comment by guedz** [comment]
Thanks for your feedback. I understand your position.
Best regards
